src/libs/actions/Policy.js

```diff
@@ -714,13 +714,14 @@ function updateWorkspaceCustomUnitAndRate(policyID, currentCustomUnit, newCustom
         },
     ];
 
+    const newCustomUnitRateParam = _.omit(newCustomUnit.rates, ['pendingAction', 'errors']);
     API.write(
         'UpdateWorkspaceCustomUnitAndRate',
         {
             policyID,
             lastModified,
             customUnit: JSON.stringify(newCustomUnit),
```

Comment:
I think we need to modify newCustomUnit too as it contains rates with the pendingAction errors fields